content/search_api_3/latest/searching/offers/attendance-mode.md

```diff
@@ -0,0 +1,39 @@
+---
+---
+
+# Attendance mode
+
+The attendance mode indicates whether an event takes places at a physical location, online or both. 
+
+| Value | Meaning |
+|--|--|
+| `offline` | visitors can attend the event at a physical location | 
+| `online` | visitor can only participate online | 
+| `mixed` | it is possible to attend the event both on a physical location or online | 
+
+You can filter by attendance mode using two methods:
+* URL parameter
+* Advanced queries
+
+Note that places will always have the attendance mode "offline", only events can have a different attendance mode.
```

Comment:
I am not sure a place even has this property. The code is written only for events.